  File "/scratch/tyoeasley/brain_representations/comp_Psim_mtx.py", line 42
    print(sys.argv[1] " " sys.argv[2])
                      ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/scratch/tyoeasley/brain_representations/DistMatrixFiltration.py", line 27, in <module>
    DistMatrixFiltration(sys.argv[1],sys.argv[2])
  File "/scratch/tyoeasley/brain_representations/DistMatrixFiltration.py", line 10, in DistMatrixFiltration
    distance_matrix = np.genfromtxt(fname_in,delimiter=",")
  File "/home/e.ty/.conda/envs/alg_top/lib/python3.9/site-packages/numpy/lib/npyio.py", line 1791, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "/home/e.ty/.conda/envs/alg_top/lib/python3.9/site-packages/numpy/lib/_datasource.py", line 194, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/e.ty/.conda/envs/alg_top/lib/python3.9/site-packages/numpy/lib/_datasource.py", line 531, in open
    raise IOError("%s not found." % path)
OSError: /scratch/tyoeasley/brain_representations/profumo_reps/HCP_Profumo_cifti_bigdata_1200subs_50modes_smooth_5mm_4runs.ppp/NetMats/subj_dists_NM.csv not found.
cp: cannot stat '/scratch/tyoeasley/brain_representations/profumo_reps/HCP_Profumo_cifti_bigdata_1200subs_50modes_smooth_5mm_4runs.ppp/NetMats/subj_persdiag_NM_PROFUMO.npy': No such file or directory
